hugo/content/changelog.md

```diff
@@ -6,6 +6,7 @@ lastmodifierdisplayname = "Naresh Rayapati"
 +++
```

Comment:
can you also update docs on AddComment step at  https://github.com/jenkinsci/jira-steps-plugin/tree/master/hugo/content/steps/comment please

Reply:
Of course! I will add an example.
You mean 'Assign Issue' for this pull request?